onecodex/viz/_heatmap.py

```diff
@@ -238,6 +251,24 @@ def plot_heatmap(
                 haxis, header=alt.Header(titleOrient="bottom", labelOrient="bottom")
             )
 
+        # Drop all-NaN rows, convert remaining NaNs to 0s, and concat
+        # dropped all-NaN rows to the end
+        dropped = []
+        for classification_id in all_nan_classification_ids:
+            d = df[(df == classification_id).any(axis=1)]
+            if self.__class__.__name__ == "OneCodexAccessor":
+                # the df of a `OneCodexAccessor` does not have NaNs, and we want to display
+                # all_nan_classification_id columns as white instead of color corresponding to 0
+                d = d.replace(0.0, np.nan)
+            dropped.append(d)
+            index = df[(df == classification_id).any(axis=1)].index
```

Comment:
lines 264 and 265 can be replaced with:

```python
df = df.drop(classification_id)
```

Reply:
This isn't working for me, actually. It leads to a `KeyError`

Reply:
whoops, didn't realize we were using a `long` format df instead of `wide`